schema.ts

```diff
@@ -274,6 +274,11 @@ export interface SliderSchema extends BaseNumberSchema<ValueHintType.Slider> {
   step?: number | string;
 }
 
+/**
+ * Icons that can be used with a {@link ScaleSchema}.
+ *
+ * For example, to render a star rating, use a {@link ScaleSchema} with `icon: ScaleIconSet.Star`.
```

Comment:
Nitpick: Can you prefix the enum with `coda.`, since that's the default in the style guide.

Reply:
Done.